examples/resources/github.yaml

```diff
@@ -15,7 +15,8 @@ spec:
   display: Github
   # mapping of Github team memberships to Teleport cluster roles
   teams_to_logins:
-    - organization: <github-org>
-      team: <github-team>
-      logins:
-        - "admin"
+  - logins:
+    - access
+    - admin
```

Comment:
let's not use `admin` role - it's too permissive. Instead please use `editor` role.

Reply:
I've updated `github.yaml` and all other OSS Connector in examples